content/chainguard/chainguard-images/working-with-images/fips-images.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "FIPS-enabled Chainguard Images"
+title: "Chainguard's Kernel Independent FIPS-Enabled Images"
```

Comment:
Can we use the actual product title here? "Chainguard FIPS Images" is the term used on the website, order sheets, legal documents, contracts, all other marketing?